app/utilities/protocol_importers/protocols_io/v3/protocol_normalizer.rb

```diff
@@ -7,6 +7,9 @@ class ProtocolNormalizer < ProtocolImporters::ProtocolNormalizer
         def normalize_protocol(client_data)
           # client_data is HttpParty ApiReponse object
           protocol_hash = client_data.parsed_response.with_indifferent_access[:protocol]
+          unless protocol_hash.present?
+            raise NormalizerError.new(:nil_protocol, 'Protocol not present in hash.')
```

Comment:
This is using different behaviour as the `ApiClient`. We could split the `NormalizerError` into:
- InvalidJSON
- ParseError (for missing/incorrect values)

If fields are optional and do not raise an error, we should add `&` to them or check for existence, e.g. `protocol_hash[:image][:source]`.